test_conformance/spirv_new/main.cpp

```diff
@@ -174,9 +175,40 @@ test_status checkAddressWidth(cl_device_id id)
   return TEST_PASS;
 }
 
+void printUsage() {
+    log_info("Test spirv_new read binary spv files from default path spirv_bin\n");
```

Comment:
I would avoid encoding the test name here and quote the default path for readability and compose the string from the default value.

`Reading SPIR-V files from default '%s' path.`